### PR TITLE
 Issue #456: synchronizes eclipse-cs and maven 

### PR DESCRIPTION
**Pull request by rnveach**
Issue #456

https://github.com/checkstyle/eclipse-cs/issues/456#issuecomment-1407569510
This resolves everything except 
> maven-checkstyle is set to look for folder structures like src\main\resources

I wasn't sure if we wanted to make such a change in eclipse. I look at egit/jgit, which is another eclipse plugin, and it doesn't do this. I am not sure if eclipse projects prefer to keep this way, unlike normal maven projects.

### CODE REVIEW

File: net.sf.eclipsecs.checkstyle/metadata/com/puppycrawl/tools/checkstyle/checkstyle-metadata_1_0.dtd [hunk 1]
```diff
@@ -8,138 +8,138 @@
 -->
```
Comment (rnveach) [inline net.sf.eclipsecs.checkstyle/metadata/com/puppycrawl/tools/checkstyle/checkstyle-metadata_1_0.dtd:8]: @Calixte What is the purpose of https://github.com/checkstyle/eclipse-cs/tree/master/docs and how is it updated?

I noticed it has the same DTD files at https://github.com/checkstyle/eclipse-cs/tree/master/docs/dtds that I updated in this PR, but nothing required those files to be updated as well.

Comment (Calixte) [reply net.sf.eclipsecs.checkstyle/metadata/com/puppycrawl/tools/checkstyle/checkstyle-metadata_1_0.dtd:8]: The `/docs` directory is the directory configured to be served by GitHub Pages at https://checkstyle.org/eclipse-cs/
It is automatically generated during the release when running `mvn package` on `net.sf.eclipsecs.doc`.
I believe the two are decoupled so the source doc `net.sf.eclipsecs.doc` can be used to generate the website, but also the Eclipse Help content.
See those lines in the release script: https://github.com/checkstyle/eclipse-cs/blob/master/release.sh#L70-L75

Comment (rnveach) [reply net.sf.eclipsecs.checkstyle/metadata/com/puppycrawl/tools/checkstyle/checkstyle-metadata_1_0.dtd:8]: So basically `/docs` is the suppose to be the last release's documentation (10.4.0) as we are in snapshot for the next release. Eclipse-CS doesn't use another repo for the hosted site like main repo does at https://github.com/checkstyle/checkstyle.github.io ?

So basically https://checkstyle.org/eclipse-cs/ is a mirror of http://checkstyle.github.io/eclipse-cs/ which points to https://github.com/checkstyle/eclipse-cs/tree/master/docs/ in virtually real time?

Comment (Calixte) [reply net.sf.eclipsecs.checkstyle/metadata/com/puppycrawl/tools/checkstyle/checkstyle-metadata_1_0.dtd:8]: You are correct.

Comment (rnveach) [reply net.sf.eclipsecs.checkstyle/metadata/com/puppycrawl/tools/checkstyle/checkstyle-metadata_1_0.dtd:8]: Thanks @Calixte

Comment (Calixte) [reply net.sf.eclipsecs.checkstyle/metadata/com/puppycrawl/tools/checkstyle/checkstyle-metadata_1_0.dtd:8]: > nothing required those files to be updated as well

Actually, those fail when I build now.

**Pull request merged**